Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4391,12 +4391,32 @@ ObservableList<ControlProperty> IWindowExplorer.GetElementProperties(ElementInfo
 
             if (el != null)
             {
-                //Learn optional values
-                if (ElementInfo.ElementTypeEnum == eElementType.ComboBox || ElementInfo.ElementTypeEnum == eElementType.List)
+                if (ElementInfo.ElementTypeEnum == eElementType.TextBox || ElementInfo.ElementTypeEnum == eElementType.Text ||
+                    ElementInfo.ElementTypeEnum == eElementType.ComboBox || ElementInfo.ElementTypeEnum == eElementType.ComboBoxOption ||
+                    ElementInfo.ElementTypeEnum == eElementType.List || ElementInfo.ElementTypeEnum == eElementType.ListItem)
                 {
-                    foreach (IWebElement value in el.FindElements(By.XPath("*")))
-                        ElementInfo.OptionalValues.Add(value.Text);
-                    list.Add(new ControlProperty() { Name = "Optional Values", Value = ElementInfo.OptionalValuesAsString });
+                    bool isDefault = false;
+                    foreach (IWebElement val in el.FindElements(By.XPath("*")))
+                    {
+                        if (!string.IsNullOrEmpty(val.Text))
+                        {
+                            string[] tempOpVals = val.Text.Split('\n');
```

Comment:
now that we removed span and div from supported elementTypes for optional values, do we need this split based on "\n" i worried that if user will have possible value with multiple rows we will break it. 
what do you think?